src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorWrapper.java

```diff
@@ -205,13 +217,8 @@ public RawPacket reverseTransform(RawPacket pkt)
             {
                 usingAbsoluteSendTime = true;
 
-                this.rbe = new RemoteBitrateEstimatorAbsSendTime(observer);
-                if (logger.isTraceEnabled())
-                {
-                    logger.trace("created_rbe," + hashCode()
-                            + "," + System.currentTimeMillis()
-                            + "," + this.rbe.hashCode());
-                }
+                rbe = new RemoteBitrateEstimatorAbsSendTime(
```

Comment:
formatting

Reply:
How would you like me to reformat the line?